### PR TITLE
Ticket 106297680

### PR DESCRIPTION
**Pull request by ljesmin**

### CODE REVIEW

File: attributes/default.rb [hunk 1]
```diff
@@ -23,3 +31,13 @@
 when "rhel"
   default['cog_security']['ssh_service_name'] = "sshd"
 end
+
+default['fail2ban']['services'] = {
+  'ssh' => {
+    "enabled" => "true",
+    "port" => "ssh",
+    "filter" => "sshd",
+    "logpath" => '/var/log/messages',
```
Comment (whysthatso) [inline attributes/default.rb]: as this file is not actually parsed, but only a target of rsyslog, would it make sense to log to syslog here?

File: recipes/default.rb [hunk 2]
```diff
@@ -67,17 +68,39 @@
   end
 end
```
Comment (whysthatso) [inline recipes/default.rb]: for below: why not manage the whole sshd_config file?

Comment (ljesmin) [inline recipes/default.rb]: Debian (raspberries), Centos and Amazon Linux - i haven't checked incompatibilities between openssh configurations.

Comment (whysthatso) [inline recipes/default.rb]: ok

File: attributes/default.rb (hunk 1 above)
Comment (ljesmin) [inline attributes/default.rb]: This is not log destination, this is log source where fail2ban ingests information about failed logins. 

It should log it's own actions via syslog.

Comment (whysthatso) [inline attributes/default.rb]: ok

**Comment by whysthatso** [comment]
all good

**Pull request merged**